utils/build.ps1

```diff
@@ -1043,6 +1129,10 @@ function Build-Compilers() {
         LLVM_NATIVE_TOOL_DIR = $BuildTools;
         LLVM_TABLEGEN = (Join-Path $BuildTools -ChildPath "llvm-tblgen.exe");
         LLVM_USE_HOST_TOOLS = "NO";
+        Python3_EXECUTABLE = "$python";
+        Python3_INCLUDE_DIR = "$BinaryCache\Python$($Arch.CMakeName)-$PythonVersion\tools\include";
+        Python3_LIBRARY = "$BinaryCache\Python$($Arch.CMakeName)-$PythonVersion\tools\libs\python39.lib";
+        Python3_ROOT_DIR = "$BinaryCache\Python$($Arch.CMakeName)-$PythonVersion\tools";
         SWIFT_BUILD_SWIFT_SYNTAX = "YES";
```

Comment:
I notice you are cross-compiling the new swift-syntax parser too. Do you have further CMake patches to make that work (similar to my #71552), pass in the extra flags manually, or is this disabled elsewhere?

Reply:
Or is this just cross-compilation support to build SDKs, not to cross-compile the toolchain itself?

Reply:
This is cross-compiling the toolchain and we did not find a need to add any additional flags.

Reply:
> This is cross-compiling the toolchain

OK

> we did not find a need to add any additional flags.

No, I looked into it now and [you were passing in extra SDK flags manually before](https://github.com/apple/swift/pull/71584/files#diff-c861caa2fcc08744108e542ca836b07e342f537566e8c5f7a21a0e9125331ed9L1036) and [are still doing so now](https://github.com/apple/swift/blob/dab2f44b1d4a4c59d97fe518b8106d8457ceae36/utils/build.ps1#L798), as that's the only way this cross-compile works. You should try my cross-compilation pull out with your cross-compilation workflow, as I set the flags for a freshly-built cross-compilation SDK whereas it appears you manually pass in the extra flags for a prebuilt Windows ARM cross-compilation SDK. I think both should work fine together, as externally supplied flags should override the internal ones, but it would be good to verify that.

Reply:
@finagolfin Can you elaborate what "try my cross-compilation pull out with your cross-compilation workflow" means? How exactly do we "verify that"? I'd be happy to run or test something.

Reply:
Thanks, @hjyamauchi. Simply apply my patch from #71552 to your Swift source when you are cross-compiling the Swift toolchain for Windows ARM.

Alternately, check your cross-compilation log for which bootstrapping mode you are using on Windows. If it doesn't say [`--   Bootstrapping:  CROSSCOMPILE` like this (the linked log is when cross-compiling the toolchain for Android AArch64 from a linux x86_64 host)](https://github.com/finagolfin/termux-packages/actions/runs/7868262902/job/21465201357#step:8:9387), then you should be fine and there's no need to test my patch, as my patch only applies to `CROSSCOMPILE` mode. I think that is the mode you're using, but I don't know what other modifications you've made.

Reply:
@finagolfin I reran the cross-compile build with your patch https://github.com/apple/swift/pull/71552 and it was successful with no issues. The log also didn't say "CROSSCOMPILE". So it seems all good.

Reply:
Great, what bootstrap mode are you using though?

Reply:
We don't use any bootstrap mode. We use the host tool.

Reply:
OK, that is one of the bootstrapping modes, as [you can see on the official linux CI, `--   Bootstrapping:  HOSTTOOLS`](https://ci.swift.org/job/swift-PR-Linux-smoke-test/11535/consoleText). Yes, technically that doesn't make sense, but that was the wrong name that was chosen for this internal build variable, instead of something accurate like "Swift compiler source compilation mode."

Reply:
Gotcha